src/pages/workspaces/workspace/analysis/utils/app-utils.ts

```diff
@@ -34,6 +34,13 @@ export const doesWorkspaceSupportCromwellApp = (workspaceCreatedDate: string, cl
   )
 }
 
+export const getAppStatusForDisplay = (status: AppStatus): DisplayAppStatus => Utils.switchCase(status,
```

Comment:
Would it make sense to use appStatuses for this? Like, `appStatuses.find(({ status: statusType }) => status === statusType)?.displayStatus`?